S3/English/ASS_32.ass

```diff
@@ -0,0 +1,111 @@
+Video Zoom Percent: 0.375000
+Scroll Position: 36
+Active Line: 52
+Video Position: 4808
+
+[V4+ Styles]
+Format: Name, Fontname, Fontsize, PrimaryColour, SecondaryColour, OutlineColour, BackColour, Bold, Italic, Underline, StrikeOut, ScaleX, ScaleY, Spacing, Angle, BorderStyle, Outline, Shadow, Alignment, MarginL, MarginR, MarginV, Encoding
+Style: Default,Arial,45,&H00FFFFFF,&H000000FF,&H00000000,&H00000000,0,0,0,0,100,100,0,0,1,2,2,2,18,0,20,1
+Style: Title,Arial,75,&H006CC2E9,&H000000FF,&H00000000,&H00000000,0,0,0,0,100,100,0,0,1,2,2,2,10,10,280,1
+
+[Events]
+Format: Layer, Start, End, Style, Name, MarginL, MarginR, MarginV, Effect, Text
+Dialogue: 0,0:01:02.51,0:01:05.56,Title,,0,0,0,, {\pos(970,920)} {\b1\an5\pos(970,880)\fs120\frz4.5\t(0,100,1.5\fs45)}Episode 32:  {\c&H7990E7&}How much surprise that I didn't know{\b0}
+Dialogue: 0,0:01:15.13,0:01:16.70,Default,,0,0,0,,Oh, why didn't Ira come
+Dialogue: 0,0:01:17.30,0:01:18.70,Default,,0,0,0,,He'll be damaged by the sun
+Dialogue: 0,0:01:19.93,0:01:20.45,Default,,0,0,0,,Next
+Dialogue: 0,0:01:22.19,0:01:23.51,Default,,0,0,0,,I'll be with Lily next
+Dialogue: 0,0:01:24.04,0:01:24.95,Default,,0,0,0,,Should I sit in the front
+Dialogue: 0,0:01:25.54,0:01:26.82,Default,,0,0,0,,Don't be afraid, hold me close
+Dialogue: 0,0:01:27.00,0:01:27.50,Default,,0,0,0,,Neil
+Dialogue: 0,0:01:27.78,0:01:29.10,Default,,0,0,0,,Or should I sit in the back
+Dialogue: 0,0:01:29.10,0:01:30.25,Default,,0,0,0,,Don't worry, I'm here
+Dialogue: 0,0:01:30.81,0:01:31.33,Default,,0,0,0,,Neil
+Dialogue: 0,0:01:32.42,0:01:33.30,Default,,0,0,0,,So hard to decide...
+Dialogue: 0,0:01:33.35,0:01:33.73,Default,,0,0,0,,Neil
+Dialogue: 0,0:01:34.00,0:01:34.50,Default,,0,0,0,,Neil
+Dialogue: 0,0:01:35.25,0:01:36.15,Default,,0,0,0,,We're next
+Dialogue: 0,0:01:37.97,0:01:38.80,Default,,0,0,0,,Let's go
+Dialogue: 0,0:01:42.04,0:01:42.75,Default,,0,0,0,,Next
+Dialogue: 0,0:01:42.95,0:01:43.60,Default,,0,0,0,,Here
+Dialogue: 0,0:01:54.39,0:01:55.10,Default,,0,0,0,,It's awesome!
+Dialogue: 0,0:01:55.10,0:01:56.70,Default,,0,0,0,,Let's do this again. Luis
+Dialogue: 0,0:01:58.39,0:01:58.80,Default,,0,0,0,,Luis?
+Dialogue: 0,0:01:59.64,0:02:01.60,Default,,0,0,0,,Somebody help! Somebody died
+Dialogue: 0,0:02:01.76,0:02:02.84,Default,,0,0,0,,Luis
+Dialogue: 0,0:02:11.63,0:02:12.67,Default,,0,0,0,,Lynn
+Dialogue: 0,0:02:13.44,0:02:15.67,Default,,0,0,0,,Could you go to another pool for a moment
+Dialogue: 0,0:02:15.90,0:02:16.40,Default,,0,0,0,,What's the matter
+Dialogue: 0,0:02:16.90,0:02:19.60,Default,,0,0,0,,You and Lily had turned all the water in the pool into holy water
+Dialogue: 0,0:02:20.70,0:02:21.90,Default,,0,0,0,,So where should we go then
+Dialogue: 0,0:02:24.64,0:02:25.90,Default,,0,0,0,,I got an idea, come with me
+Dialogue: 0,0:02:40.19,0:02:41.50,Default,,0,0,0,,Can't you swim, Neil
+Dialogue: 0,0:02:42.23,0:02:42.84,Default,,0,0,0,,Lily
+Dialogue: 0,0:02:44.90,0:02:46.00,Default,,0,0,0,,I couldn't learn it
```

Comment:
"I never learned" would be better imo

Reply:
No. He meant he did actually tried but failed to learn